arch/arm/src/cxd56xx/Kconfig

```diff
@@ -80,7 +80,7 @@ if CXD56_PMIC
 config CXD56_PMIC_INT
 	bool "PMIC interrupt"
 	default n
-	depends on SCHED_LPWORK
```

Comment:
@xiaoxiang781216 could you please write more about this change and changes in other files `depends on SCHED_HPWORK` -> `depends on SCHED_WORKQUEUE`. I see that both `SCHED_LPWORK` and `SCHED_HPWORK` are defaulted to `n`, so I'm wondering how things are getting working now if for example `CXD56_PMIC_INT` is selected together with `SCHED_WORKQUEUE`, but `SCHED_LPWORK` is `n`.

Reply:
> @xiaoxiang781216 could you please write more about this change and changes in other files `depends on SCHED_HPWORK` -> `depends on SCHED_WORKQUEUE`. I see that both `SCHED_LPWORK` and `SCHED_HPWORK` are defaulted to `n`, so I'm wondering how things are getting working now if for example `CXD56_PMIC_INT` is selected together with `SCHED_WORKQUEUE`, but `SCHED_LPWORK` is `n`.

SCHED_WORKQUEUE can't be selected in defconfig since it doesn't have a prompt string:
https://github.com/apache/incubator-nuttx/blob/master/sched/Kconfig#L1581-L1582
On the other hand, SCHED_WORKQUEUE is selected by SCHED_LPWORK or SCHED_HPWORK automatically:
https://github.com/apache/incubator-nuttx/blob/master/sched/Kconfig#L1601
https://github.com/apache/incubator-nuttx/blob/master/sched/Kconfig#L1667
So, if SCHED_WORKQUEUE is enable, which mean at least one of SCHED_LPWORK and SCHED_HPWORK is enabled too.